lms/static/sass/multicourse/_course_about.scss

```diff
@@ -439,6 +439,13 @@
           background: url('#{$static-path}/images/link-icon.png') left center no-repeat;
         }
       }
+      &.about-sidebar-html {
```

Comment:
I think these styles are not needed
```
position: relative;
clear: both;
margin-top: 50px;
```

Reply:
@caesar2164 Did you try removing these? Can we do without?

Reply:
removed